course/assets/js/div_toggle.js

```diff
@@ -0,0 +1,21 @@
+export const initDivToggle = () => {
+  const revealElements = document.querySelectorAll("[class*=reveal]")
+  for (const reveal of revealElements) {
+    (function() {
```

Comment:
I'm not sure we need this IIFE here since all of the variables below have block scope